mapper/types.go

```diff
@@ -60,6 +71,11 @@ var (
 		Decimals: 18,
 	}
 
+	AtomicAvaxCurrency = &types.Currency{
```

Comment:
@bilgehansahin I don't quite get why we have this. Why can't we just use `AvaxCurrency` defined above?

Reply:
We use this to represent amounts in nanoAVAX (see the `decimals: 9` below and 18 in AvaxCurrency above). It makes it easier to use Rosetta for non C-chain transactions rather than converting everyting into weis back and forth.

Reply:
We use AtomicAvaxCurrency in P-chain and C-chain atomic transactions. AvaxCurrency should only be used in C-chain transactions.